AlfrescoApp/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -93,84 +93,84 @@
 "sync.downloadsize.prompt.cancel" = "Don't Sync";
 "sync.downloadsize.prompt.confirm" = "Sync Now";
 "sync.enable.title" = "Sync Files";
-"sync.enable.message" = "Would you like to automatically keep your favorite files in sync with this %@?";
+"sync.enable.message" = "Automatically keep your favorite files in sync with this %@?";
```

Comment:
I don't think this string will to be visible to the user any more and can be removed; as we'll no longer track favorites as the sync set.